api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -30,6 +31,8 @@
 import org.pmiops.workbench.db.dao.ConceptSetDao;
 import org.pmiops.workbench.db.dao.UserDao;
 import org.pmiops.workbench.db.dao.UserService;
+import org.pmiops.workbench.exceptions.*;
```

Comment:
Nit: I prefer (and I think established convention has been) to import specific exceptions rather than `*`.